faraday-net_http.gemspec

```diff
@@ -25,5 +25,5 @@ Gem::Specification.new do |spec|
   spec.files = Dir.glob('lib/**/*') + %w[README.md LICENSE.md]
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'net-http'
+  spec.add_runtime_dependency 'net-http', '>= 0.5.0'
```

Comment:
This version limitation creates an issue when gems are installed at runtime. The standard version is lower than the required one, leading to two different versions being loaded simultaneously. This causes the following error:

```
Unable to activate faraday-net_http-3.4.0, because net-http-0.4.1 conflicts with net-http (>= 0.5.0) (Gem::ConflictError)
```

The workaround was to keep using the previous version of `faraday-net-http`. 

😄

Reply:
This is also what we [state in the `gemspec`](https://github.com/lostisland/faraday-net_http/blob/main/faraday-net_http.gemspec#L18), so I'd go with option 1 unless I'm missing something important?